src/pkey/clu_pkey.c

```diff
@@ -154,6 +188,81 @@ static int wolfCLU_pKeytoPubKey(WOLFSSL_EVP_PKEY* pkey, unsigned char** out)
     return ret;
 }
 
+/* creates an out buffer containing the private key from the pkey
+ * returns size of buffer on success
+ */
+static int wolfCLU_pKeytoPriKey(WOLFSSL_EVP_PKEY* pkey, unsigned char** out)
+{
+    int type;
+    int ret = 0;
+
+    type   = wolfSSL_EVP_PKEY_id(pkey);
+    switch (type) {
+        case EVP_PKEY_RSA:
+            ret = wolfSSL_i2d_RSAPrivateKey(
+                    wolfSSL_EVP_PKEY_get0_RSA(pkey), out);
+            break;
+
+        case EVP_PKEY_DSA:
+            WOLFCLU_LOG(WOLFCLU_L0, "DSA key not yet supported");
+            ret = USER_INPUT_ERROR;
+            break;
+
+        case EVP_PKEY_EC:
+            {
```

Comment:
`{` not needed, but if using, would prefer to use it on all `case`s.

Reply:
I seem to remember an old issue with something like C89 and a windows build being where I picked up using the { in a case statement in order to make sure the declared variables were at the top of the scope. Will tidy up the code more here.